content/docs/started/getting-started-icp.md

```diff
@@ -71,47 +71,47 @@ NAME         STATUS    ROLES                      AGE       VERSION
     The PVs defination file (`pv.yaml`) is as following:
 
      ```yaml
```

Comment:
The YAML file doesn't render properly. See preview:
https://deploy-preview-1038--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-icp/

Reply:
Oh sorry, I see it just now, I'm checking...